ui/admin/app/templates/application.hbs

```diff
@@ -132,33 +132,40 @@
 {{/if}}
 
 <PendingConfirmations as |confirmation accept deny|>
-  <Rose::Dialog
-    @heading={{if
-      confirmation.options.title
-      (t confirmation.options.title)
-      (t 'actions.confirm')
-    }}
-    @dismissButtonText={{t 'actions.cancel'}}
-    @icon='flight-icons/svg/alert-triangle-16'
-    @style='warning'
-    @modal={{true}}
-    @dismiss={{deny}}
-    as |dialog|
-  >
-    <dialog.body>
-      <p>{{confirmation.text}}</p>
-    </dialog.body>
-    <dialog.footer>
-      <Rose::Button @style='primary' {{on 'click' accept}}>
-        {{if
-          confirmation.options.confirm
-          (t confirmation.options.confirm)
-          (t 'actions.ok')
-        }}
-      </Rose::Button>
-      <Rose::Button @style='secondary' {{on 'click' deny}}>
-        {{t 'actions.cancel'}}
-      </Rose::Button>
-    </dialog.footer>
-  </Rose::Dialog>
+  <Hds::Modal @color='warning' as |M|>
+    <M.Header
+      @dismissButtonText={{t 'actions.cancel'}}
+      @modal={{true}}
+      @dismiss={{deny}}
```

Comment:
It looks like it's called `@onDismiss` in [HDS](https://helios.hashicorp.design/utilities/dialog-primitive?tab=code#dialogprimitiveheader)

Reply:
Thanks for pointing to the docs. Removed 👍

Reply:
We should still keep it, since it's still valid functionality

Reply:
Even if removing does not change functionality?

Reply:
The `deny` function won't get called when you hit the dismiss button if it's removed, so it likely still does the same thing right now as `deny` also just closes but technically the cancel button and dismiss button now do different things

Reply:
Thanks 👍